bootstrap/src/main/config/mediaserver.xml

```diff
@@ -47,7 +47,10 @@
 	<resources>
 		<localConnection poolSize="100" />
 		<remoteConnection poolSize="50" />
-		<player poolSize="50" />
+		<player>
+			<poolSize>50</poolSize>
+			<cacheSize>100</cacheSize>
```

Comment:
can you add some comments about this value? Is this something we can fine tune given a certain traffic load in the server?